app/app_main.c

```diff
@@ -453,23 +453,9 @@ static int enable_aes(ACVP_CTX *ctx) {
     CHECK_ENABLE_CAP_RV(rv);
     rv = acvp_cap_sym_cipher_set_parm(ctx, ACVP_AES_GCM, ACVP_SYM_CIPH_IVLEN, 96);
     CHECK_ENABLE_CAP_RV(rv);
-    rv = acvp_cap_sym_cipher_set_parm(ctx, ACVP_AES_GCM, ACVP_SYM_CIPH_PTLEN, 16);
+    rv = acvp_cap_sym_cipher_set_domain(ctx, ACVP_AES_GCM, ACVP_SYM_CIPH_PTLEN, 0, 65536, 8);
```

Comment:
May want to increase the increment value from 8 to 256. I'm not sure if NIST will actually try every PT or AAD length in the domain, but with an increment of 8 that would be 8192 PT and AAD sizes. AT 256 it's only 256 iterations max and could be a noticeable difference in test time.